app/src/main/java/com/owncloud/android/ui/activity/SettingsActivity.java

```diff
@@ -410,6 +421,45 @@ private void setupRecommendPreference(PreferenceCategory preferenceCategoryMore)
         }
     }
 
+    private void setupE2EPreference(PreferenceCategory preferenceCategoryMore) {
+        Preference preference = findPreference("setup_e2e");
+
+        if (preference != null) {
+            if (FileOperationsHelper.isEndToEndEncryptionSetup(this, user) ||
+                !CapabilityUtils.getCapability(this).getEndToEndEncryptionKeysExist().isFalse()) {
```

Comment:
```suggestion
                CapabilityUtils.getCapability(this).getEndToEndEncryptionKeysExist().isTrue()) {
```

Otherwise it will also be hidden for UNKNOWN value.

Reply:
I thought that this is what it should be…and I was wrong :S